app/v1/validation/abbreviatedReturn/AbbreviatedReturnValidator.scala

```diff
@@ -120,3 +124,9 @@ case class NegativeAngieError(amt: BigDecimal) extends Validation {
   val path = JsPath \ "angie"
   val value = Json.toJson(amt)
 }
+
+case class AngieDecimalError(amt: BigDecimal) extends Validation {
+  val errorMessage: String = "ANGIE has greater than the allowed 2 decimal places."
```

Comment:
This error messages need to feed from messages.en

Reply:
messages.en doesn't exist on API